packages/twenty-server/jest.config.ts

```diff
@@ -10,7 +8,25 @@ const jestConfig: JestConfigWithTsJest = {
   transformIgnorePatterns: ['/node_modules/'],
   testRegex: '.*\\.spec\\.ts$',
   transform: {
-    '^.+\\.(t|j)s$': 'ts-jest',
+    '^.+\\.(t|j)s$': [
```

Comment:
Jest uses babel out of the box for transpilation, I switched it to SWC which we've already done on twenty-front, twenty-ui and twenty-shared. This should improve performance and is more consistent with the normal build process.